doc/backup.md

```diff
@@ -0,0 +1,122 @@
+# Database Backup and Restore Script
+
+## Overview
+
+This manual describes steps to perform for backup creation and its restoration.
+
+## Key Functions
+
+- **Backup**: Creates a timestamped `tar.gz` archive of the PostgreSQL database and Django media files.
+- **Restore**: Restores from a specified `tar.gz` backup archive.
+
+## Docker Compose Integration
+
+The `backup` service in `docker-compose.yaml` manages backup and restore using the `scripts/db.sh` script.
+
+### Configuration
+
+All variables can be set in the [environment file](.env.template). These include variables necessary for connection to the database, path to local folder where the backup is created and setup of cron tasks for backup.
+
+- **Volumes**:
+  - `${BACKUP_VOLUME:-../backups}:/backups`
+  - `.:/code`
+
+### Operations
+
+#### Enable Automatic Backups
+
+To ensure automatic backups are enabled, set `ENABLE_BACKUPS=true` (enabled by default):
+
+```bash
+ENABLE_BACKUPS=true docker compose up -d backup
+```
+
+This will configure automatic backups based on the `BACKUP_SCHEDULE`.
+
+To disable automatic backups:
+
+```bash
+ENABLE_BACKUPS=false docker compose up -d backup
+```
+
+#### Manual Backup
+
+Create a manual backup:
+
+```bash
+docker compose exec backup sh /code/scripts/db.sh backup
+```
+
+- **Output**: `backup_<timestamp>.tar.gz` in the `BACKUP_DIR` (`../backups` by default).
+
+## Scheduled Backup with Cron
+
+To schedule the backup script to run automatically at a specific time using cron, add the following line to your crontab:
+
+1. Open the crontab editor:
+
+    ```bash
+    crontab -e
```

Comment:
is this supposed to be system cron or cron inside the container?

Reply:
yes the cron job is in the VM so it can stop and start docker-compose with no issue